modules/javafx.graphics/src/main/java/com/sun/glass/ui/Screen.java

```diff
@@ -41,7 +40,9 @@
     private static final int dpiOverride;
 
     static {
-        dpiOverride = AccessController.doPrivileged((PrivilegedAction<Integer>) () -> Integer.getInteger("com.sun.javafx.screenDPI", 0)).intValue();
+        @SuppressWarnings("removal")
```

Comment:
Combine assignment and declaration?

Reply:
Good idea. This allows the static block to be removed.